EventListener/SerializerListener.php

```diff
@@ -3,6 +3,8 @@
 namespace XApi\LrsBundle\EventListener;
 
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Serializer\Exception\ExceptionInterface as BaseSerializerException;
```

Comment:
We shouldn't rely on the Symfony Serializer component, but on the xAPI serializer API. However, this can be done later as the current implementation of the `SerializerListener` already doesn't follow this approach.